docs-next/components/Page.tsx

```diff
@@ -13,110 +14,119 @@ import { TableOfContents } from './TableOfContents';
 
 export const Page = ({
   headings = [],
+  title,
   children,
   isProse,
 }: {
   headings?: Heading[];
   children: ReactNode;
+  title?: string;
   isProse?: boolean;
 }) => {
   const [mobileNavCollapsed, setMobileNavCollapsed] = useState(true);
   const contentRef = useRef<HTMLDivElement | null>(null);
+  const metaTitle = title ? `Keystone Next Documentation - ${title}` : `Keystone Next`;
```

Comment:
nit: Can this be swapped around? Ie; `${title} - Keystone Next Documentation`. Reason: The browser tab shows the title, but if I have multiple docs pages open, it's impossible to know which is which because the phrase` Keystone Next Documentation` fills the entire tab width.

Reply:
good call, thnx Jess 🎉